carball/analysis/kickoff_detection/kickoff_analysis.py

```diff
@@ -0,0 +1,159 @@
+                kPlayer.player.id = player.id.id
+                kPlayer.kpos = BaseKickoff.get_kickoff_pos(player, data_frame, frame)
+                kPlayer.tpos = BaseKickoff.get_touch_pos(player, data_frame, frame, end_frame)
+                kPlayer.boost = data_frame[player.name]['boost'][end_frame]
+                BaseKickoff.set_jumps(kPlayer, player, data_frame, frame, end_frame)
+            cur_kickoff.type = BaseKickoff.get_kickoff_type(cur_kickoff.touch.players)
+            kickoffs[frame] = cur_kickoff
+        return kickoffs
+
+    @staticmethod
+    def set_jumps(kPlayer, player, data_frame, frame, end_frame):
+        ja  = data_frame[player.name]['jump_active']
+        dja = data_frame[player.name]['double_jump_active']
+        time = 0.0
+        firstBall = True
+        # check the kickoff frames (and then some) for jumps & big boost collection
+        for f in range(frame, end_frame + 20):
+            time = time + data_frame['game']['delta'][f]
+            if(firstBall and BaseKickoff.get_dist(data_frame, player.name, f) < 700):
+                kPlayer.ballTime = time
+                firstball = False
+            if (data_frame[player.name]['boost_collect'][f] == True):
+                kPlayer.boostTime = time
+            if (ja[f] != ja[f-1] or dja[f] != dja[f-1]):
+                kPlayer.jumps.append(time)
+
+    @staticmethod
+    def get_kickoff_type(players: list):
+        #
+        diagonals = [player.kpos for player in players].count(0)
+        offcenter = [player.kpos for player in players].count(1)
+        goalies   = [player.kpos for player in players].count(2)
+        if (len(players) == 6):
+            # 3's
+            if (diagonals == 4):
+                if (offcenter == 2):
+                    return kickoff.DDO
+                if (goalies == 2):
+                    return kickoff.DDG
+            if (diagonals == 2):
+                if (offcenter == 4):
+                    return kickoff.DOO
+                if (offcenter == 2):
+                    return kickoff.DOG
+            if (offcenter == 4):
+                return kickoff.OOG
+        if (len(players) == 4):
+            if (diagonals == 4):
+                return kickoff.DD
+            if (diagonals == 2):
+                if (offcenter == 2):
+                    return kickoff.DO
+                if (goalies == 2):
+                    return kickoff.DG
+            if (offcenter == 4):
+                return kickoff.OO
+            if (offcenter == 2):
+                if (goalie == 2):
+                    return kickoff.OG
+        if (len(players) == 2):
+            if (diagonals == 2):
+                return kickoff.D
+            if (offcenter == 2):
+                return kickoff.O
+            if (goalies == 2):
+                return kickoff.G
+        return kickoff.U
+
+    @staticmethod
+    def get_kickoff_pos(player_class: Player, data_frame: pd.DataFrame, frame: int):
```

Comment:
Let's do position instead of pos in method names.  I believe that is what we are doing elsewhere.